en/releases/centreon-commercial-extensions.md

```diff
@@ -34,6 +34,14 @@ commercial extension. No worry, the compatibility with the previous licensing me
 
 ## Centreon BAM
 
+### 20.04.1
+
+* The default calculation method is now "Worst" when you create a business activity
+* Business activities configuration: You're now able to only save warning threshold
+* Reporting: not 24x7 timeperiod management in charts & tables
```

Comment:
This may be confusing into "no 24x7 timeperiod management in charts and tables" whereas it is the opposite actually.
Suggested : 24x7 timeperiods are now properly handled in charts and tables